docs/helm/rbac-values.yaml

```diff
@@ -0,0 +1,12 @@
+# Contains account e-mails and namespaces as needed for the rolebinding templates.
+ClusterAdmins::
+ - YOUR_GCP_LOGIN_EMAIL
+
+# Replace NamespaceName and GCPServiceAccount with your own values.
+# Add more/less NamespaceName and GCPServiceAccount pairs pertaining to your use case.
+# Following values gives GCPServiceAccount permissions to deploy to NamespaceName.
+Permissions:
+ - NamespaceName: "test"
```

Comment:
Suggest replacing this with a variable as well: YOUR_CLUSTER_NAMESPACE

Reply:
Done